test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -449,18 +449,22 @@ def setUpClass(cls):
         cls.init_param()
         cls.check_users_exist()
         cls.init_servers()
+        cls.init_schedulers()
         if cls.use_cur_setup:
             _, path = tempfile.mkstemp(prefix="saved_custom_setup",
                                        suffix=".json")
-            ret = cls.server.save_configuration(path, 'w')
+            ret = cls.server.save_configuration()
+            if not ret:
+                cls.logger.error("Failed to save custom setup")
```

Comment:
Can we change the log message to `Failed to save Server's custom setup`? Same for other places.